ci-operator/config/openshift/svt/openshift-svt-master__regression-aws-4.19-nightly-x86.yaml

```diff
@@ -140,6 +140,22 @@ tests:
     test:
     - ref: openshift-svt-regression
     workflow: openshift-qe-installer-aws
+- always_run: false
+  as: replace-loadedworker-ocp-40557
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "30"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.2xlarge
+      KUBE_BURNER_VERSION: defalt
```

Comment:
@qiliRedHat can we correct from `KUBE_BURNER_VERSION: defalt` to `KUBE_BURNER_VERSION: default`?

Reply:
fixed the typo and started a new test.